aleemer/Internify
# Feat/in110 update techreqs

#### User Story
IN-110 Update Technical Requirements Page with appropriate key checkboxes
https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1/backlog?selectedIssue=IN-110

#### Changes made
Within Create.js, the mockTechStackData object has been changed so that the languages, frameworks, worktools, and cs concepts have more comprehensive lists of items.  The lists are alphabetized, and formatted to be consistent with how they appear in the student objects.

In addition, custom textboxes have been removed (courtesy of @aleemer )

UPDATE: languages, frameworks, worktools, cs concepts now align with the survey

#### Does your new code introduce new warnings on dev console?
No.

#### Test Steps
Run the website, attempt to create a new job posting, and will notice there are many new languages/frameworks/etc to choose from
